.pre-commit-config.yaml

```diff
@@ -0,0 +1,26 @@
+repos:
+  - repo: https://github.com/psf/black
+    rev: 24.3.0
+    hooks:
+      - id: black
+        args: ["--target-version", "py38"]
+  - repo: https://github.com/PyCQA/isort
+    rev: 5.13.2
+    hooks:
+      - id: isort
+        args: ["--profile", "black"]
+  - repo: https://github.com/pre-commit/mirrors-prettier
+    rev: v3.1.0
+    hooks:
+      - id: prettier
+        args: [--prose-wrap=preserve, --print-width=90]
+  - repo: https://github.com/pycqa/flake8
+    rev: 7.0.0
+    hooks:
+      - id: flake8
+        args: ["--max-line-length=90", "--extend-ignore=E203,W503"]
```

Comment:
Why 90 here? Black is 88 by default, right? (that's my standard)

Reply:
unfortunately there are lines that are up to 90 chars long which cannot be simply shorten. 
default flake8 limit is 79
it can be disabled like other rules, but i usually leave it enabled, it forces contributors to write shorter and more on point ...

Reply:
Yeah, 88 is pretty much a new standard thanks to black. I just don't get why we would set it to 90 just because one specific line in the current code can't be autoformatted to 88 (and happens to be 90) :) We can just fix that manually or add `# noqa`. Anyway I'll fix that separately 👍